eaw-pglib-pgdebug-emmyluadoc.lua

```diff
@@ -0,0 +1,59 @@
+-- ------------------------------------------------------------------
+-- ------------------------------------------------------------------
+--  _____                _                  _     _    _
+-- |  ___|              (_)                | |   | |  | |
+-- | |__ _ __ ___  _ __  _ _ __ ___    __ _| |_  | |  | | __ _ _ __
+-- |  __| '_ ` _ \| '_ \| | '__/ _ \  / _` | __| | |/\| |/ _` | '__|
+-- | |__| | | | | | |_) | | | |  __/ | (_| | |_  \  /\  / (_| | |
+-- \____/_| |_| |_| .__/|_|_|  \___|  \__,_|\__|  \/  \/ \__,_|_|
+--                | |
+--                |_|
+--  _____                          _                ______
+-- |  ___|                        | |               |  _  \
+-- | |__ _ __ ___  _ __ ___  _   _| |    _   _  __ _| | | |___   ___
+-- |  __| '_ ` _ \| '_ ` _ \| | | | |   | | | |/ _` | | | / _ \ / __|
+-- | |__| | | | | | | | | | | |_| | |___| |_| | (_| | |/ / (_) | (__
+-- \____/_| |_| |_|_| |_| |_|\__, \_____/\__,_|\__,_|___/ \___/ \___|
+--                            __/ |
+--                           |___/
+-- ------------------------------------------------------------------
+-- ------------------------------------------------------------------
+---Opens a message box that contains the handled event and its parameters as multi-line formatted string.
+---**WARNING:** Appears to be non-functional.
+---@param event any TODO: This is most likely be the same Event type as used in gamescoring.lua
+---@param params table
+function DebugEventAlert(event, params) end
+
+---@vararg string
+---Opens a message box that contains the formatted input string provided as a vararg.
+---The vararg should be a string with _n_ formatting placeholders, and _n_ elements that will be handled via `string.format(unpack(arg))`.
+---**WARNING:** Appears to be non-functional!
```

Comment:
It works in the debug exe if it is started without the ignore asserts option

Reply:
I dind't know that worked, but then again I've never gotten the AI debug window to work either - maybe it's a similar issue where it works for some. But I'll update that accordingly.